syntax = "proto3";

message PollRegistration{
    string poll_registration_id = 1;
    string id_user = 2;
    string name = 3;
}

message PollRegistrationContainer {
    repeated PollRegistration entries = 1;
}
